package.json

```diff
@@ -23,16 +23,21 @@
   "dependencies": {
     "cors": "^2.8.4",
     "express-async-handler": "^1.1.3",
+    "express-query-boolean": "^2.0.0",
```

Comment:
Is this package essential? I believe the bedrock-account-http package would function sufficiently without the need of converting strings to booleans, we should be weary pulling in outside packages that are not necessary.

Reply:
@gannan08 I believe this is a good solution.  The issue is that a query parameter `?foo=true` shows up in express as a string, and this throws a wrench in the works when one would like to validate the query parameter as a boolean using json-schema/bedrock-validation.  

Perhaps the *best* solution is to have our own implementation of this included in bedrock-express, but these things take time and haggling.

Reply:
> this throws a wrench in the works when one would like to validate the query parameter as a boolean using json-schema/bedrock-validation.

Okay, so we convert expected parameters to booleans to check against our validator. That is cool with me.

> Perhaps the _best_ solution is to have our own implementation of this included in bedrock-express, but these things take time and haggling.

Agreed, we're definitely not at a point where we can spend time on haggling things like this. Don't know if we want to file an issue to come back to this later or add a FIXME in the code.

Reply:
added a FIXME to the route and require.